components/notability/wikis/osu/notability_checker_config.lua

```diff
@@ -0,0 +1,321 @@
+---
+-- @Liquipedia
+-- wiki=osu
+-- page=Module:NotabilityChecker/config
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Table = require('Module:Table')
+
+local Config = {}
+
+
+-- These are constants, you don't need to touch them
+-- unless values for liquipediatiertype change
+Config.TIER_TYPE_GENERAL = 'general'
+Config.TIER_TYPE_QUALIFIER = 'qualifier'
+Config.TIER_TYPE_WEEKLY = 'weekly'
+Config.TIER_TYPE_MONTHLY = 'monthly'
+Config.TIER_TYPE_SHOW_MATCH = 'show match'
+
+-- How many placements should we retrieve from LPDB for a team/player?
+Config.PLACEMENT_LIMIT = 2000
+
+Config.MAX_NUMBER_OF_PARTICIPANTS = 20
+Config.MAX_NUMBER_OF_COACHES = 5
+
+-- Which LPDB placement parameters do we care about?
+Config.PLACEMENT_QUERY =
+	'pagename, tournament, date, placement, liquipediatier, ' ..
+	'liquipediatiertype, players, extradata, mode'
```

Comment:
probably oos, but `players` is deprecated, we should look into getting rid of it in these modules, should use `opponentplayers` instead